metadata-ingestion/src/datahub/ingestion/source/kafka/kafka.py

```diff
@@ -164,6 +164,25 @@ def get_kafka_consumer(
     return consumer
 
 
+def get_kakfa_admin_client(
+    connection: KafkaConsumerConnectionConfig,
+) -> AdminClient:
+    client = AdminClient(
+        {
+            "group.id": "test",
```

Comment:
Shouldn't this be some random group id?
`test` seems to be too generic, what if it already exists?

Reply:
In this case, we actually do not consume any messages so any group id is fine, I think.

Reply:
I think this should work as it is datahub-specific, so there is less chance to collide